lib/main.js

```diff
@@ -118,6 +118,12 @@ function logServerPort (address, port) {
   logger.info(logMessage);
 }
 
+function initHeapdump (args) {
+  if (args.isHeapdumpEnabled) {
```

Comment:
i wouldn't find this particularly unreadable to just include the 3 lines in the main function below. but i don't feel strongly about it.